src/MessageHandler/ImageHandler.php

```diff
@@ -42,9 +43,10 @@ public function __invoke(ImageMessage $message): void
             }
         }
 
-        // @todo: send message to geo-encoder
-
-        // @todo: create next message
-        throw new UnrecoverableMessageHandlingException('Not implemented yet');
+        if (!is_null($message->getEventId())) {
+            $this->messageBus->dispatch(new GeocoderMessage($message->getEventId()));
```

Comment:
Why are we geocoding inside an image handler? Is “an image” something else than a picture in this case?